src/elements/ProjectsMonitoringCounters/projectsMonitoringCounters.scss

```diff
@@ -21,18 +27,44 @@
       font-size: 1rem;
     }
 
+    &__placeholder-subtitle{
+      height: 24px;
+    }
+
+    &__header-icon {
+      height: 20px;
+      width: 20px;
+      margin-right: 5px;
+
+      path {
+        fill: $ceriseRed;
+      }
+    }
+
     &__counter {
       display: flex;
       flex: 1;
       min-height: 58px;
       margin: 0 0 5px 0;
       color: $primary;
       font-weight: 700;
-      font-size: 42px;
+      font-size: 38px;
       align-items: center;
       justify-content: center;
     }
 
+    &__link {
+      color: $darkPurple;
+      cursor: pointer;
+
+      &:hover {
+        text-decoration: underline;
+        color: $bombay;
+        text-underline-offset: 1px;
+        text-decoration-thickness: 2px
```

Comment:
missed semicolon. Also please use style-lint to fix CSS file

Reply:
Done